src/OpenLoco/Windows/BuildVehicle.cpp

```diff
@@ -391,6 +395,11 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
             });
     }
 
+    static bool vehicleIsLocked(CompanyId companyId, uint16_t vehicleObjIndex)
```

Comment:
identical function already exists at `Company::isVehicleIndexUnlocked()`, use that instead.